client/landing/gutenboarding/controller.tsx

```diff
@@ -19,12 +18,7 @@ export const redirectIfNotEnabled: PageJS.Callback = ( context, next ) => {
 	next();
 };
 
-export const hideMasterbar: PageJS.Callback = ( context, next ) => {
-	context.store.dispatch( hideMasterbarAction() );
-	next();
-};
-
 export const main: PageJS.Callback = ( context, next ) => {
-	context.primary = <Gutenboard />;
+	context.layout = <Gutenboard />;
```

Comment:
`context.layout` gets rendered directly to `#wpcom`. This allows us to omit the actual `<Layout />` component (and masterbar).